### ISSUE TITLE
ctrl+alt+del does not seem to log me out

### ISSUE THREAD

**Issue by gantryyork** [pop-os/cosmic-epoch#187]
- ctrl+alt+del does not seem to log me out of bring up the pop-up dialog
- Choosing log out from the applet brings up a pop-up dialog that we must confirm.  Usually this takes two clicks on confirm
-

**Comment by jokeyrhyme** [comment]
Yeah, I, too, have noticed that sometimes clicks don't seem to register

**Comment by jokeyrhyme** [comment]
There's currently no Ctrl+Alt+Del binding in the default config.ron: https://github.com/pop-os/cosmic-comp/blob/master_jammy/config.ron

**Comment by jokeyrhyme** [comment]
Okay, it's possible to add a keybinding that calls `busctl --user call com.system76.CosmicSession /com/system76/CosmicSession com.system76.CosmicSession Exit` (similar the the existing ones for changing screen brightness)

However, this bypasses the confirmation dialog, which may or may not be desirable

**Comment by git-f0x** [comment]
The log out shortcut is <kbd>Super</kbd> + <kbd>Shift</kbd> + <kbd>Escape</kbd>. So this can probably be closed.

**Comment by leviport** [comment]
Super + Shift + Escape seems to be working nicely for me
